GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/DehydrateTests.cs

```diff
@@ -176,279 +176,264 @@ public void DehydrateShouldFailOnWrongDiskLayoutVersion()
-            this.fileSystem.ReadAllText(fileToRead1);
-            this.fileSystem.ReadAllText(fileToRead2);
+            string childFolderToDehydrate = Path.Combine("GVFS", "GVFS.Mount");
+            string parentFolderToDehydrate = "GVFS";
+            string fileToReadInChildFolder = Path.Combine(childFolderToDehydrate, "Program.cs");
+            string fileToReadInParentFolder = Path.Combine(parentFolderToDehydrate, "GVFS.UnitTests", "Program.cs");
+            string fileToReadInChildFolderVirtualPath = this.Enlistment.GetVirtualPathTo(fileToReadInChildFolder);
+            string fileToReadInChildFolderBackingPath = this.Enlistment.GetBackingPathTo(fileToReadInChildFolder);
+            string fileToReadInParentFolderVirtualPath = this.Enlistment.GetVirtualPathTo(fileToReadInParentFolder);
+            string fileToReadInParentFolderBackingPath = this.Enlistment.GetBackingPathTo(fileToReadInParentFolder);
+            this.fileSystem.ReadAllText(fileToReadInChildFolderVirtualPath);
+            this.fileSystem.ReadAllText(fileToReadInParentFolderVirtualPath);
 
             this.DehydrateShouldSucceed(
-                new[] { $"{folderToDehydrate1} {FolderDehydrateSuccessfulMessage}", $"{folderToDehydrate2} {FolderDehydrateSuccessfulMessage}" },
+                new[] { $"{childFolderToDehydrate} {FolderDehydrateSuccessfulMessage}", $"{parentFolderToDehydrate} {FolderDehydrateSuccessfulMessage}" },
                 confirm: true,
                 noStatus: false,
-                foldersToDehydrate: string.Join(";", folderToDehydrate1, folderToDehydrate2));
+                foldersToDehydrate: string.Join(";", childFolderToDehydrate, parentFolderToDehydrate));
 
             this.Enlistment.UnmountGVFS();
 
-            fileToRead1.ShouldNotExistOnDisk(this.fileSystem);
-            fileToRead2.ShouldNotExistOnDisk(this.fileSystem);
+            fileToReadInChildFolderBackingPath.ShouldNotExistOnDisk(this.fileSystem);
+            fileToReadInParentFolderBackingPath.ShouldNotExistOnDisk(this.fileSystem);
         }
 
-        [TestCase]
+        [TestCase]
         public void FolderDehydrateNestedFoldersParentBeforeChild()
         {
-            string folderToDehydrate1 = "GVFS";
-            string folderToDehydrate2 = Path.Combine("GVFS", "GVFS.Mount");
-            string fileToRead1 = this.Enlistment.GetVirtualPathTo(Path.Combine(folderToDehydrate1, "GVFS.UnitTests", "Program.cs"));
-            string fileToRead2 = this.Enlistment.GetVirtualPathTo(Path.Combine(folderToDehydrate2, "Program.cs"));
-            this.fileSystem.ReadAllText(fileToRead1);
-            this.fileSystem.ReadAllText(fileToRead2);
+            string parentFolderToDehydrate = "GVFS";
+            string childFolderToDehydrate = Path.Combine("GVFS", "GVFS.Mount");
+            string fileToReadInParentFolder = Path.Combine(parentFolderToDehydrate, "GVFS.UnitTests", "Program.cs");

```

Comment:
Same comment here: The variable is named `fileToReadInParentFolder` but the file is not in the parent folder.